java/AccStats.java

```diff
@@ -367,6 +367,6 @@ public static int countStuckVals(
         }
         return output;
     }
-   
+
 
 }
```

Comment:
missing end of line :D 
Good to add this in even though it is not part of your changes :D

Reply:
I see. In fact my editor added this but I think I deleted it.

Reply:
Usually we should have a newline char at the end of a file :D 

https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file provides a bit more insights.